java/ql/src/semmle/code/java/dataflow/FlowSources.qll

```diff
@@ -270,3 +271,39 @@ class AndroidIntentInput extends DataFlow::Node {
     )
   }
 }
+
+/**
+ * Method access to external inputs of `android.content.Intent` object
+ */
+class IntentGetExtraMethodAccess extends MethodAccess {
+  IntentGetExtraMethodAccess() {
+    exists(AndroidComponent ac |
+      this.getEnclosingCallable().getDeclaringType() = ac and ac.isExported()
+    ) and
+    (
+      this.getMethod().getName().regexpMatch("get\\w+Extra") and
+      this.getMethod().getDeclaringType() instanceof TypeIntent
+      or
+      this.getMethod().getName().regexpMatch("get\\w+") and
+      this.getQualifier().(MethodAccess).getMethod().hasName("getExtras") and
+      this.getQualifier().(MethodAccess).getMethod().getDeclaringType() instanceof TypeIntent
+    )
+  }
+}
+
+/**
+ * Android intent extra source
```

Comment:
```suggestion
 * Android intent extra source.
```

Reply:
Thanks @intrigus-lgtm for reviewing this PR. The text is not a complete sentence, which is the reason I didn't add a period at the end. However, to be consistent with other comments already in the same file, I've fixed the issue.